### PR TITLE
Feature/#39 Create main page

### PR DESCRIPTION
**Pull request by FFreestyler**

### CODE REVIEW

**Comment by FFreestyler** [comment]
[Create main page](https://trello.com/c/dn7upwYC/39-create-main-page)

**Review by MangriMen — APPROVED** [review]

**Comment by FFreestyler** [comment]
image

File: src/components/pages/Mainpage.tsx [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+  const error = useSelector(selectError);
+
+  useEffect(() => {
+    if (error) {
+      alert(error);
+    }
+  }, [error]);
+
+  const [checked] = useState(true);
```
Comment (StepanovRoma) [inline src/components/pages/Mainpage.tsx]: useState do nothing

File: src/components/common/Navbar.tsx [hunk 2]
```diff
@@ -77,6 +77,12 @@ const Navbar: FC = () => {
     setAnchorElUser(null);
   };
 
+  useEffect(() => {
+    if (!isLogged) {
+      navigate(ROUTE.AUTH);
+    }
+  }, [isLogged, navigate]);
+
```
Comment (StepanovRoma) [inline src/components/common/Navbar.tsx]: It is better to do this using private routing

**Review by StepanovRoma — CHANGES_REQUESTED** [review]

**Review by MangriMen — CHANGES_REQUESTED** [review]
Rename Mainpage to MainPage

**Review by MangriMen — APPROVED** [review]

**Review by StepanovRoma — APPROVED** [review]

**Pull request merged**